cmake/k4aCompilerFlags.cmake

```diff
@@ -29,6 +29,11 @@ if ("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
         # Added in clang 5
         list(APPEND CLANG_ALL_WARNINGS "-Wno-zero-as-null-pointer-constant") # Allow zero as nullptr
     endif()
+    if (NOT (${CMAKE_CXX_COMPILER_VERSION} VERSION_LESS "8.0.0"))
```

Comment:
Took me a second to parse what this was trying to accomplish. Consider:

`if (${CMAKE_CXX_COMPILER_VERSION} VERSION_GREATER_EQUAL "8.0.0")`

Reply:
It looks like we support CMake 3.5.1 (which comes with Ubuntu 16.04), `VERSION_GREATER_EQUAL` was not added until CMake 3.7, which is why this check was done this way for the above 5.0.0 check as well.

Reply:
Ah... I missed that. Sounds good.